includes/Classifai/Providers/OpenAI/ChatGPT.php

```diff
@@ -84,6 +85,48 @@ public function render_provider_fields() {
 			]
 		);
 
+		switch ( $this->feature_instance::ID ) {
+			case TitleGeneration::ID:
+				add_settings_field(
+					'number_of_titles',
```

Comment:
In thinking about this some more, both of these settings in essence do the same thing. I'm thinking we can consolidate this down to a single setting that is used by both Features. Maybe change the ID to be `static::ID . '_number_of_suggestions'`, can change the label to be something like `Number of suggestions` and can also change the description to be more generic, like `Number of suggestions that will be generated in one request`